### DISCUSSION TITLE
Cannot connect anymore

### DISCUSSION THREAD

**Discussion by Cloud-Kid** [haugene/docker-transmission-openvpn#2885] [Q&A] [answered]
Hi,

My setup was working fine until my Nord VPN subscription failed, I changed my password, re-subscibed for a suscription, edit my password environment variable with my new password hash thing and now I can't execute the container properly anymore. I'm greated with this error : 
```
Starting container with revision: 07f5a2b9aea5028c9bb75438c1552708e91dde71
TRANSMISSION_HOME is currently set to: /config/transmission-home
Creating TUN device /dev/net/tun
Using OpenVPN provider: NORDVPN
Running with VPN_CONFIG_SOURCE auto
Provider NORDVPN has a bundled setup script. Defaulting to internal config
Executing setup script for NORDVPN
/etc/openvpn/nordvpn/..
INFO: OVPN: Checking curl installation
INFO: OVPN: DNS resolution ok
INFO: OVPN: ok, configurations download site reachable
INFO: OVPN: Removing existing configs in /etc/openvpn/nordvpn
parse error: Invalid numeric literal at line 1, column 7
```
Is it password related or is it about something else ?

Regards, Cloudkid

**Comment by Cloud-Kid** [comment]
The issue resolved itself on the next day, may be because of a delay in NordVPN password application. Resolved.

**Reply by edgd1er** [reply, marked as answer]
most of the time, that error `parse error: Invalid numeric literal at line 1, column 7` is the result of the API not responding as expected: 
html when json is expected.
the cause is your ip being blacklisted  due to too many restarts of the container.
the solution, stop the container, wait for a while, restart

**Reply by Cloud-Kid** [reply]
Thanks for your input, it definitely was that !
